### PR TITLE
topic-2

### PR DESCRIPTION
**Pull request by sBrink-worth**
Second set of exercises ready to be reviewed.

### CODE REVIEW

File: Topic-2-CSS/exercise-1.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+html {
+  font-family: 'Source Sans Pro', Helvetica, Arial;
+  font-size: 14px;
+}
```
Comment (maximilianotulian) [inline Topic-2-CSS/exercise-1.css]: Nice style, just keep in mind that classes are always better. Do you know the cause of that?

File: Topic-2-CSS/exercise-2-A.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+html {
+    font-family: 'Source Sans Pro', Helvetica, Arial;
+    font-size: 14px;
+}
+  
+.aside {
```
Comment (maximilianotulian) [inline Topic-2-CSS/exercise-2-A.css]: +1

File: Topic-2-CSS/exercise-4-B.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
+@media (min-width: 600px){
+
+    main {
+        display: grid;
+        grid-template-columns: 1fr 1fr 1fr 1fr 1fr;
```
Comment (maximilianotulian) [inline Topic-2-CSS/exercise-4-B.css]: **Minor**: Try to keep the same level of Indentation across the files

File: Topic-2-CSS/exercise-4-B.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            <img src="https://media.gettyimages.com/photos/raven-picture-id145572609?s=2048x2048" class="figure-img" alt="crow3">
+        </figure> 
+        <figure class="figure4">
+            <img src="https://media.gettyimages.com/photos/raven-with-open-wings-flying-on-blue-sky-picture-id172794920?s=2048x2048" class="figure-img" alt="crow4">
+        </figure>
+        <figure class="figure5">
+            <img src="https://media.gettyimages.com/photos/crows-flying-picture-id184978905?s=2048x2048" class="figure-img alt="crow5">
+        </figure>
+        <figure class="figure6">
+            <img src="https://media.gettyimages.com/photos/awesome-black-raven-picture-id514470635?s=2048x2048" class="figure-img" alt="crow6">
```
Comment (maximilianotulian) [inline Topic-2-CSS/exercise-4-B.html]: Do you know what is the difference if the file is in the src?

File: Topic-2-CSS/exercise-1.css (hunk 1 above)
Comment (sBrink-worth) [reply Topic-2-CSS/exercise-1.css]: Yeah! Classes can be re-usable.

**Pull request merged**